test/cpp/naming/resolver_component_tests_runner_invoker.cc

```diff
@@ -123,24 +123,30 @@ int main(int argc, char** argv) {
         bin_dir + "/resolver_test_record_groups.yaml",
         bin_dir + "/utils/dns_resolver", bin_dir + "/utils/tcp_connect");
 #else
-    grpc::testing::ManifestFile manifest_file(
-        grpc::testing::NormalizeFilePath(test_srcdir.value()) + "\\MANIFEST");
-    grpc::testing::InvokeResolverComponentTestsRunner(
+#ifndef GRPC_PORT_ISOLATED_RUNTIME
+    gpr_log(GPR_ERROR,
```

Comment:
add a comment that you're knowingly misusing GRPC_PORT_ISOLATED_RUNTIME as an indication whether you're running on RBE or not (I think it's fine to use it, but the code should ack that this define is meant for something else and it just came handy in this situation). I think you can treat this as a TODO for finding a better way of detecting whether you're on RBE or not.

Reply:
Done.